tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -64,6 +64,15 @@ function install_artifacts() {
 	chmod +x /opt/kata/bin/*
 	[ -d /opt/kata/runtime-rs/bin ] && \
 		chmod +x /opt/kata/runtime-rs/bin/*
+
+	# Allow Mariner to use custom configuration.
+	if [ "${HOST_OS}" == "cbl-mariner" ]; then
+		config_path="/opt/kata/share/defaults/kata-containers/configuration-clh.toml"
+		clh_path="/opt/kata/bin/cloud-hypervisor-glibc"
+		sed -i -E 's|(enable_annotations) = .+|\1 = ["initrd", "kernel"]|' "${config_path}"
+		sed -i -E "s|(valid_hypervisor_paths) = .+|\1 = [\"${clh_path}\"]|" "${config_path}"
+		sed -i -E "s|(path) = \".+/cloud-hypervisor\"|\1 = \"${clh_path}\"|" "${config_path}"
```

Comment:
if we're doing this here, then we could as well specify the mariner kernel and initrd here instead of annotations?

Reply:
I'm only doing this here because these are impossible to set via annotations. I like `setup.sh` since it can also be cleanly reused for other environments.